openapi/resource-server.yaml

```diff
@@ -576,14 +612,23 @@ paths:
                     id: 'https://openpayments.guide/alice/quotes/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
```

Comment:
should the quote request body still include `sendAmount` or be changed to `maxSendAmount`?
Even though it doesn't match `maxSendAmount` I _think_ `sendAmount` is good to leave as is

Reply:
I was thinking about this as well while working on https://github.com/interledger/rafiki/issues/1639. I think it might be confusing to have `sendAmount` going in an `maxSendAmount` coming back. The user might not understand they are the same thing and wonder if `maxSendAmount` is derived or transformed from `sendAmount` instead of set directly. If they are the same thing they should probably have the same name.

Reply:
It's all `debitAmount` now, even in the grant